task/06-conditions-n-loops-tasks.js

```diff
@@ -30,7 +30,14 @@
  *
  */
 function getFizzBuzz(num) {
-    throw new Error('Not implemented');
+    if (num % 15 == 0)
+        return "FizzBuzz";
+    else if (num % 3 == 0)
+        return "Fizz";
+    else if (num % 5 == 0)
```

Comment:
"operator === is always preferable
sometimes ''==' may cause an error
(check at console :
"0" == 0
and "0" === 0)"